src/app/objectdetails/objectdetails.component.html

```diff
@@ -5,6 +5,11 @@
 <div *ngIf="_mediaObject as obj" style="padding:10px;">
     <app-objectviewer #objectviewerComponent [mediaobject]="obj"></app-objectviewer>
```

Comment:
It seems a waste of screen estate to me, that the toggle in which way this is sorted is not _right_ of the `app-objectviewer`.